build-source-tarball.sh

```diff
@@ -98,6 +104,11 @@ while :; do
     shift
 done
 
+# if [ $BASE_TARBALL -eq 1 ] && [[ '$CUSTOM_SDK_DIR' != ''] || [ '$CUSTOM_REF_PACKAGES_DIR' != '' ] || [ '$CUSTOM_PREVIOUSLY_BUILT_PACKAGES_DIR' != '' ]]; then
```

Comment:
Why is this commented out?

Reply:
This was a WIP that I didn't finish.  It should not allow building a base tarball with the --with* parameters.  I've fixed it.